MAINTAINERS.yml

```diff
@@ -6424,6 +6424,7 @@ nRF Platforms:
     - anangl
     - masz-nordic
     - bjarki-andreasen
+    - carlescufi
```

Comment:
Mmmh, do you really need _four_ maintainers? This is really too many IMO, and something we've typically been trying to avoid.

Reply:
Ideally yes, because we do have quite a lot of PRs coming in from multiple teams at Nordic, and different PRs typically need different reviewers. If you don't mind, this would make our lives easier.

Reply:
Right, that's what I am saying. The current "nRF Platform" entry covers a lot of ground, and depending on the area touched one or another maintainer is required. So if the solution is to split it up I wonder if there are any precedents of splitting up a platform entry in maintainers so I can take some inspiration from.